.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+# Syntax can be found here: https://docs.github.com/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
+
+@Charles-Gagnon @lucyzhang929 @chlafreniere
```

Comment:
It's interesting how syntax highlighting only highlights the first line. This is the right format for this file, right? (just making sure 😄)

Reply:
Should be based on the doc - guess we'll find out in the next PR 😁